src/include/libabrt.h

```diff
@@ -52,6 +52,8 @@ void ensure_writable_dir_group(const char *dir, mode_t mode, const char *user, c
 char *run_unstrip_n(const char *dump_dir_name, unsigned timeout_sec);
 #define get_backtrace abrt_get_backtrace
 char *get_backtrace(const char *dump_dir_name, unsigned timeout_sec, const char *debuginfo_dirs);
+#define get_sub_components abrt_get_sub_components
```

Comment:
Please don’t do this. Just declare the function as `abrt_get_sub_components()`, otherwise it will be impossible to debug.

Reply:
That's what I'm thinking every time I run into this. What is the point of these `#define`s exactly?  
  
Anyway, I'll move the whole thing to libreport, I'm just asking generally.

Reply:
I have no idea, but I hate it every time when gdb doesn’t understand me when I try to add a breakpoint.

Reply:
Wouldn't it make sense to get rid of those altogether then?

Reply:
Yeah, I think their only purpose is to be gotten rid of. If you’re sufficiently tickled by this, I absolutely welcome such changes.